types/dragino/lsn50v2-d20/alarm.schema.json

```diff
@@ -0,0 +1,52 @@
+{
+  "$id": "https://akenza.io/dragino/lsn50v2-d20/alarm.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Alarm",
+  "type": "object",
+  "properties": {
+    "c1Temperature": {
+      "title": "Channel 1 temperature",
+      "unit": "°C",
+      "type": "number",
+      "description": "The temperature in °C.",
+      "minimum": -3276.5,
+      "maximum": 3276.5
+    },
+    "c1tempMin": {
+      "title": "Temperature minimum",
+      "unit": "°C",
+      "type": "number",
+      "description": "The Channel 1 temperature minimum."
+    },
+    "c1tempMax": {
+      "title": "Temperature maximum",
+      "unit": "°C",
+      "type": "number",
+      "description": "The Channel 1 temperature maximum."
+    },
+    "shtTempMin": {
+      "title": "SHT Temperature minimum",
+      "unit": "°C",
+      "type": "number",
+      "description": "The sht temperature minimum."
+    },
+    "shTempMax": {
```

Comment:
shtTemperatureMax?
possibly explain sht in the description and apply to all the other data keys

Reply:
SHT is a sensor Chip/Module https://sensirion.com/media/documents/BD45ECB5/61642783/Sensirion_Humidity_Sensors_SHT1x_Datasheet.pdf

Should i change it to "The SHT Sensor chip temperature minimum." ?